### PR TITLE
Update ISSUE_TEMPLATE to avoid general questions

### PR DESCRIPTION
**Pull request by sazzad16**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE [hunk 1]
```diff
@@ -1,3 +1,12 @@
+<!--
+Github issues SHOULD BE USED to report bugs and for DETAILED feature requests. 
+Everything else belongs in the [Jedis Google Group](https://groups.google.com/g/jedis_redis)
```
Comment (chayim) [inline .github/ISSUE_TEMPLATE:3]:
```
Discussions belong in the [Jedis Google Group](https://groups.google.com/g/jedis_redis) 
or [Jedis Github Discussions](https://github.com/redis/jedis/discussions). Discussions that belong in Google groups will be summarily closed.
```

**Review by chayim — CHANGES_REQUESTED** [review]
Thoughts?

File: .github/ISSUE_TEMPLATE (hunk 1 above)
Comment (sazzad16) [reply .github/ISSUE_TEMPLATE:3]: I prefer what we already have.

**Review by chayim — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE [hunk 2]
```diff
@@ -1,3 +1,12 @@
+<!--
+Github issues SHOULD BE USED to report bugs and for DETAILED feature requests.
```
Comment (sazzad16) [inline .github/ISSUE_TEMPLATE:2]:
```suggestion
Github issues should be used to report bugs and for detailed feature requests. 
```

**Review by chayim — APPROVED** [review]

**Pull request merged**